### PR TITLE
Bookmarklet add

### PR DESCRIPTION
**Pull request by jessyurko**

### CODE REVIEW

**Comment by ebensing** [comment]
@jessyurko Sorry, I've been out of town. I do have some questions on this though and some code to clean up before I push to production. Should be able to take care of it today.

**Comment by jessyurko** [comment]
sounds good -- let me know if you want to meet up / chat briefly to go over
it.

On Tue, Sep 10, 2013 at 10:41 AM, E.J. Bensing notifications@github.comwrote:

> Sorry, I've been out of town. I do have some questions on this though and
> some code to clean up before I push to production. Should be able to take
> care of it today.
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/Japan-Digital-Archives/Japan-Digital-Archive/pull/614#issuecomment-24165078
> .

## 

Jessica Yurkofsky

http://www.jessyurko.com
508.688.2206

File: src/JDA/CoreBundle/Resources/views/Widget/dropboxwelcome.widget.html.twig [hunk 1]
```diff
@@ -0,0 +1,23 @@
+
```
Comment (ebensing) [inline src/JDA/CoreBundle/Resources/views/Widget/dropboxwelcome.widget.html.twig:1]: What is this page for?

File: src/JDA/CoreBundle/Resources/views/Widget/duplicate.widget.html.twig [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{# src/Zeega/\CoreBundle\/Resources/views/Widget/duplicate.widget.html.twig #}
```
Comment (ebensing) [inline src/JDA/CoreBundle/Resources/views/Widget/duplicate.widget.html.twig:1]: what is this page for?

File: src/JDA/CoreBundle/Resources/views/Widget/facebookWelcome.widget.html.twig [hunk 3]
```diff
@@ -0,0 +1,24 @@
+
```
Comment (ebensing) [inline src/JDA/CoreBundle/Resources/views/Widget/facebookWelcome.widget.html.twig:1]: again, what do we use this page for?

File: src/JDA/CoreBundle/Resources/views/Widget/fail.widget.html.twig [hunk 4]
```diff
@@ -0,0 +1,39 @@
+
```
Comment (ebensing) [inline src/JDA/CoreBundle/Resources/views/Widget/fail.widget.html.twig:1]: probably sounding a bit repetitive here, but what's this page for?

**Comment by ebensing** [comment]
@jessyurko Let me know about the above stuff, also, I just went through and did some pretty big stylistic changes that it would be great if you can keep in mind on the next round of code. It's this commit: 249c5915671c6a702d655c7b19fb88d3b54305bd

**Pull request merged**